api/lib/infrastructure/repositories/student-repository.js

```diff
@@ -0,0 +1,38 @@
+const _ = require('lodash');
+const Student = require('../../domain/models/Student');
+const Bookshelf = require('../bookshelf');
+
+module.exports = {
+
+  _toStudents(results) {
+    return results.reduce((studentArray, result) => {
+      const nationalStudentId = result.nationalStudentId;
+      const student = _.find(studentArray, (student) => student.nationalStudentId === nationalStudentId);
+      const account = { userId: result.userId, certificationCount: result.certificationCount, updatedAt: result.updatedAt };
```

Comment:
On pourrait utiliser [_.pick](https://lodash.com/docs/4.17.15#pick)
```suggestion
      const account = _pick(result, [ 'userId', 'certificationCount', 'updatedAt']);
```